change language in verify docs to specify 0.5.0 or higher (#1636)
OffchainLabs/arbitrum-docs: arbitrum-docs/stylus/how-tos/verifying-contracts.mdx

---

:::caution

> This page will walk you through how to verify your Stylus contracts locally. However, Stylus contract verification is available on [Arbiscan](https://arbiscan.io/) as well. Please note, however, that Stylus contract verification on Arbiscan is only supported for Stylus contracts deployed using `cargo-stylus` 0.5.0 or higher. 

Reviewer: ```suggestion
This page will walk you through how to verify your Stylus contracts locally. Stylus contract verification is also available on [Arbiscan](https://arbiscan.io/). Please note, however, that Stylus contract verification on Arbiscan is only supported for Stylus contracts deployed using `cargo-stylus` 0.5.0 or higher. 
```
